digits/dataset/images/classification/test_views.py

```diff
@@ -66,7 +66,7 @@ class BaseViewsTestWithImageset(BaseViewsTest):
     IMAGE_WIDTH     = 10
     IMAGE_CHANNELS  = 3
     BACKEND         = 'lmdb'
-    COMPRESSION     = 'none'
+    COMPRESSION     = 'none' 
```

Comment:
Yeah, there's a lot of this. I use [this plugin](https://github.com/bronson/vim-trailing-whitespace) (shameless plug), which makes the whitespace issues super obvious.